test/reconstruct-test.js

```diff
@@ -0,0 +1,613 @@
+const vows = require('vows');
+const assert = require('assert');
+const ret = require('../dist');
+const reconstruct = require('../dist/reconstruct').reconstruct;
+const types = require('../dist/types').types;
+
+const similarToWhitespaceSet = [
+  { type: types.CHAR, value: 9 },
+  { type: types.CHAR, value: 10 },
+  { type: types.CHAR, value: 11 },
+  { type: types.CHAR, value: 12 },
+  { type: types.CHAR, value: 13 },
+  { type: types.CHAR, value: 32 },
+  { type: types.CHAR, value: 160 },
+  { type: types.CHAR, value: 5760 },
+  { type: types.RANGE, from: 8192, to: 8200 },
+  { type: types.CHAR, value: 8232 },
+  { type: types.CHAR, value: 8233 },
+  { type: types.CHAR, value: 8239 },
+  { type: types.CHAR, value: 8287 },
+  { type: types.CHAR, value: 12288 },
+  { type: types.CHAR, value: 65279 },
```

Comment:
import this value, and others that represent sets in tests, from `dist/sets.js`

Reply:
This is intentionally different from that set; (the range in the correct whitespace set is `{ type: types.RANGE, from: 8192, to: 8203 }`); it's there as a test case to fail with [this](https://github.com/fent/ret.js/pull/25#discussion_r551753241) bug you noted.

Reply:
ah, didn't catch that